Python/optimizer_bytecodes.c

```diff
@@ -886,6 +886,44 @@ dummy_func(void) {
         }
     }
 
+    op(_CALL_ISINSTANCE, (callable, self_or_null, args[oparg] -- res)) {
+        if (sym_is_null(self_or_null) || sym_is_not_null(self_or_null)) {
```

Comment:
I've seen this guard used elsewhere with `self_or_null` but it's not clear to me whether it is needed here as well?